client/src/components/Form/Form.js

```diff
@@ -110,7 +132,23 @@ const Form = ({ open, handleClose }) => {
                     please provide details and type of help required of your business.
                 </ContentText>
                 <Text>Name of Business</Text>
-                <Input autoFocus margin="dense" fullWidth placeholder="e.g. Hunter & Hare" />
+                <Input
+                    autoFocus
+                    margin="dense"
+                    fullWidth
+                    placeholder="e.g. Hunter & Hare"
+                    value={storeName}
+                    onChange={e => setStoreName(e.target.value)}
+                />
+                <Text>Business Icon</Text>
+                <ImageUploader
```

Comment:
Should only allow one image to be uploaded using `singleImage` property: https://github.com/jakehartnell/react-images-upload#available-options